src/devtools/client/debugger/src/components/TestInfo/TestInfo.tsx

```diff
@@ -0,0 +1,82 @@
+import { SourceLocation } from "graphql";
+import MaterialIcon from "ui/components/shared/MaterialIcon";
+import { useFetchCypressSpec } from "ui/hooks/useFetchCypressSpec";
+import { getRecordingDuration } from "ui/reducers/timeline";
+import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
+import { TestItem, TestResult } from "ui/types";
+import { getFormattedTime } from "ui/utils/timeline";
+import { selectLocation } from "../../actions/sources";
+import { getThreadContext } from "../../selectors";
+
+export default function TestInfo({
+  spec,
+  testCases,
+  result,
+}: {
+  testCases: TestItem[];
+  spec?: string;
+  result?: TestResult;
+}) {
+  const recordingDuration = useAppSelector(getRecordingDuration);
+  const cypressResults = useFetchCypressSpec();
+
+  return (
+    <div className="flex flex-col px-4 py-2 space-y-1">
+      <div className="flex justify-between">
+        <div className="flex items-center gap-1">
+          {result ? <Status result={result} /> : null}
+          <div className="font-bold">{spec}</div>
+        </div>
+        {recordingDuration ? <div>{getFormattedTime(recordingDuration)}</div> : null}
```

Comment:
This text is not inline with the test title. Adding `items-center` to L25 will do the trick

<img width="293" alt="image" src="https://user-images.githubusercontent.com/788456/197587746-37b99b0f-6674-4bb4-af8c-02734fcb0ff4.png">